# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libprelude/libprelude-1.0.0-gcc46-stddef.h.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- libprelude-1.0.0/bindings/c++/include/prelude.hxx.orig	2012-05-08 22:31:03.914693163 +0200
+++ libprelude-1.0.0/bindings/c++/include/prelude.hxx	2012-05-08 22:34:11.680959147 +0200
@@ -26,6 +26,8 @@
 #ifndef _LIBPRELUDE_PRELUDE_HXX
 #define _LIBPRELUDE_PRELUDE_HXX
 
+#include <stddef.h>
+
 #include "prelude-client.hxx"
 #include "prelude-client-easy.hxx"
 #include "prelude-connection.hxx"
